docs/source/data.rst

```diff
@@ -0,0 +1,11 @@
+.. _data-reference:
+
+Data API
+--------
+
+.. automodule:: openscm_units.data
+
+Mixtures API
+============
+
+.. automodule:: openscm_units.data.mixtures
```

Comment:
Using this
```suggestion
.. autodata:: openscm_units.data.mixtures.MIXTURES
   :annotation:
```
the value of MIXTURES is not printed in the docs, which makes them easier to read.

Reply:
Nice